source/clear-linux/guides/maintenance/swupd-search.rst

```diff
@@ -14,14 +14,14 @@ This guide assumes you:
 * Possess a basic knowledge of :ref:`swupd <swupd-guide>` 
 * Understand :ref:`how swupd differs <swupd-about>` from  
   other Linux\* distributions 
```

Comment:
Minor revision, up to you if you want to include it in this PR: "Linux" doesn't need to be attributed here, as it is attributed earlier in the text.